### ISSUE TITLE
ARM Linux build is no longer updated on the dev channel

### ISSUE THREAD

**Issue by kaendfinger** [dart-lang/sdk#25207]
https://gsdview.appspot.com/dart-archive/channels/dev/raw/latest/sdk/

This [screenshot](http://cl.ly/1S2g3d2l2P3B/Image%202015-12-10%20at%2012.09.03%20AM.png) shows what the directory listing looks like at the time of filing this issue.

The ARM Linux build was last updated on November 18th, while x64 Linux was last updated December 7th.

**Comment by zoechi** [comment]
Have you checked the concrete version directory instead of `.../latest`? When tests fail the `latest` directory isn't updated AFAIK.

**Comment by whesse** [comment]
It was also missing from the concrete version directory.  We did upload it, but then we later ran the promote script (that uploads everything else) again, to fix a problem with api-docs, and that deleted it.
We have uploaded it again, and it is in both latest/ and in 1.14.0-dev.3.0/.

It is on bleeding edge that latest/ is not updated if tests fail.  On dev/release, latest/ is always updated when we promote a dev build to public release.
